### PR TITLE
QE: Remove flattening of AppStream repos

### PR DESCRIPTION
**Pull request by NamelessOne91**
## What does this PR change?

See https://github.com/SUSE/spacewalk/issues/24753 for details.
TL:DR - with the newly added AppStream support flattening is no longer needed. 

We can include in the current CI CLM tests some scenarios to create and apply filters.

**UPDATE**

CLM feature file tested on uyuni-master-dev-acceptance-tests-NUE (CI)

IDK if it may be worth to add scenarios for a CLM Project which uses AppStream repositories and a filter for them - currently I'm just verifying we can create such filter

## GUI diff

No difference.

- [x] **DONE**

## Documentation

- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage

- Cucumber tests were removed or modified

- [x] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/24753
Port(s): 4.3 https://github.com/SUSE/spacewalk/pull/24904

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by NamelessOne91** [comment]
I don't like the way the filter checkbox is currently handled and will try to code something better before merging. 

That said, I've created [a card to look into how we manage checkboxes](https://github.com/SUSE/spacewalk/issues/24910) in the testsuite, as it seems we have a lot of ad-hoc methods around that.

**Comment by maximenoel8** [comment]
If you remove the clm feature, you probably also need to impact these constant file : 
https://github.com/uyuni-project/uyuni/blob/79e3eacce60105c98c6d92e5d080cb9e4e3abb93/testsuite/features/support/constants.rb#L422C1-L429C78
and 
https://github.com/uyuni-project/uyuni/blob/79e3eacce60105c98c6d92e5d080cb9e4e3abb93/testsuite/features/support/constants.rb#L457C1-L464C77

**Comment by NamelessOne91** [comment]
> If you remove the clm feature, you probably also need to impact the constant file

Thanks for the heads-up. Updated.

**Review by maximenoel8 — DISMISSED** [review]
LGTM, I will be happy to test on BV once merge. Do we need to backport on 4.3 ? I think it will be nice to backport because 4.3 seems to be staying for a while.

**Comment by NamelessOne91** [comment]
> LGTM, I will be happy to test on BV once merge. Do we need to backport on 4.3 ? I think it will be nice to backport because 4.3 seems to be staying for a while.

Thanks

Yes, It'd make sense to backport it. I have it ready in the linked PR.

**Review by srbarrios — APPROVED** [review]
